src/renderer/theme.store.ts

```diff
@@ -29,12 +29,8 @@ export class ThemeStore {
     { id: "kontena-light", type: ThemeType.LIGHT },
   ];
 
-  @computed get activeThemeId() {
-    return userStore.preferences.colorTheme;
-  }
-
   @computed get activeTheme(): Theme {
-    const activeTheme = this.themes.find(theme => theme.id === this.activeThemeId) || this.themes[0];
+    const activeTheme = this.themes.find(theme => theme.id === userStore.preferences.colorTheme) || this.themes[0];
```

Comment:
How are these related to this PR?

Reply:
I did this because styling is broken if the `colorTheme` in preferences isn't valid. (And we changed the names in v4)